README.md

```diff
@@ -26,22 +38,18 @@ FHIR Converter with DotLiquid engine consists of the following components:
 
 ## Using the FHIR Converter
 
-HL7 v2 to FHIR conversion templates are attached with the Converter.
-You can test the default conversion behavior of it by using the command-line tool.
-In case the default templates do not meet your requirements, you can modify the templates by yourself.
-
-### The command-line tool
+### Command-line tool
 
 The command-line tool can be used to convert a folder containing HL7 v2 messages to FHIR resources.
 Here are the parameters that the tool accepts:
 
 | Option | Name | Optionality | Default | Description |
 | ----- | ----- | ----- |----- |----- |
 | -d | TemplateDirectory | Required | | Root directory of templates. |
-| -e | EntryTemplate | Required | | Name of entry template. |
-| -c | InputDataContent | Optional| | Input data content. Please specify OutputDataFile to get the results. |
+| -e | EntryTemplate | Required | | Name of root template. |
```

Comment:
Do we need to update the option to `-r` and the option name to `RootTemplate` for customers as well? I could create another PR to update it if needed.